src/main/java/io/github/debuggyteam/wonders/util/WondersUtils.java

```diff
@@ -1,38 +1,5 @@
 package io.github.debuggyteam.wonders.util;
```

Comment:
Is there a reason this file still exists?

Reply:
I'm thinking the stuff that used to exist in the utils file got moved or will have a different implementation. Currently this is what exists in the main repo: https://github.com/DebuggyTeam/wonders/blob/1.19/src/main/java/io/github/debuggyteam/wonders/util/WondersUtils.java

Reply:
True, however in that case shouldn't it just be yeeted?

Reply:
Probably, I'll ask snooze about it on the dev chat since snooze was the one who made the PR.